bindings.go

```diff
@@ -372,10 +383,10 @@ func GetAllStagedSectors(sectorBuilderPtr unsafe.Pointer) ([]StagedSectorMetadat
 }
 
 // GetAllSealedSectors returns a slice of all sealed sector metadata for the sector builder.
-func GetAllSealedSectors(sectorBuilderPtr unsafe.Pointer) ([]SealedSectorMetadata, error) {
+func GetAllSealedSectors(sectorBuilderPtr unsafe.Pointer, performHealthchecks bool) ([]SealedSectorMetadata, error) {
```

Comment:
boolean flags are not a particularly nice api, so I think at the go level this could be two different calls `GetAllSealedSectors` (like the old version) and a new method like `GetAllSealedSectorsWithHealth`

Reply:
I just made this change, but I'm going to complain about it:

The intent of this package is to provide a Go wrapper to our Rust lib, nothing more. Each exported function in this package corresponds one-to-one with a function defined in Rust. This is the first function which doesn't follow this convention; it will surely lead to more - ultimately causing a bunch of dumb discussions about where certain bits of software "should" live (in caller's code or in this package).